src/docs/get-started/install/_path-mac-linux.md

```diff
@@ -44,6 +44,40 @@ whenever you open a new window. For example:
     $ which flutter
     ```
 
+### Update directly (Linux)
+
+In most cases, your distribution may not acquire the path you pass
+through the terminal for future sessions. To maintain permanently, in
+some cases, you will need to change the environment variables file
+directly. Remember that it requires administrator privileges.
+For example:
+
+   1. Determine the directory where you placed the Flutter SDK.
+
+   2. Locate the `etc` directory at root of the system and open `profile` file with privileges.
+
+        ```terminal
+        $ sudo nano /etc/profile
+        ```
+   3. Update the PATH string including the location of your Flutter SDK directory.
```

Comment:
string including... => path string with the...

In the example below, replace the ellipses (...) with the original path.
Make sure to add the Flutter path to the **end** of the path list.